projects/ion/src/lib/directives/dropdown/mocks/open-dropdown.component.scss

```diff
@@ -0,0 +1 @@
+@import '@angular/cdk/overlay-prebuilt.css';
```

Comment:
Is this necessary?
`@angular/cdk/overlay-prebuilt.css` already as imported in `angular.json`

Reply:
forgot that one. lol